File: TP1 java/java fede/Ejercicio 3/Inicio.java
public class Inicio {
    public static void main(String[] args) throws Exception{
        Serie s1 = new Serie("imagen", "Doom Patrol", "Nuevos episodios los jueves");
        Serie s2 = new Serie("imagen", "Planeta de Recolectores", "Nuevos episodios los jueves");
        Serie s3 = new Serie("imagen", "Ricky and Morty", "Nuevos episodios los lunes");
        Estrenos estrenos = new Estrenos("Nuevos episodios cada semana");
        estrenos.agregarSerie(s1);
        estrenos.agregarSerie(s2);
        estrenos.agregarSerie(s3);
        estrenos.imprimir();
    }
}

File: TP1/TP1 python/python fede/ejercicio 3/Inicio.py
from Serie import Serie
from Estrenos import Estrenos

s1 = Serie("image", "Doom Patrol", "Nuevos episodios todos los jueves")
s2 = Serie("image", "Planeta de Recolectores", "Nuevos episodios todos los jueves")
s3 = Serie("image", "Rick and Morty", "Nuevos episodios todos los lunes")

estrenos = Estrenos("Nuevos episodios cada semana")

estrenos.agregarSerie(s1)
estrenos.agregarSerie(s2)
estrenos.agregarSerie(s3)

estrenos.imprimir()